.github/workflows/lint.yml

```diff
@@ -75,12 +75,17 @@ jobs:
             | jq '[.tasks[] | select(.task == "codegen" and .command != "<NONEXISTENT>")] != []' || echo 'false')
           echo "codegen=$CODEGEN" >> $GITHUB_OUTPUT
 
-          HAS_RUST=$(test -f "${{ matrix.directory }}/rust-toolchain.toml" && echo 'true' || echo 'false') 
+          HAS_RUST=$([[ -f "${{ matrix.directory }}/Cargo.toml" || ${{ matrix.directory }} = "apps/hash-graph" ]] && echo 'true' || echo 'false')
```

Comment:
nothing for right now as it is definitely overengineering (although quite elegant IMO) but in the future do we maybe want to introduce a `.hash` toml file for these kind of exceptions?

Reply:
We don't want to keep that exception. It's because it was a workspace previously and all commands using this assumption, still.